src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/posthearingreview/PostHearingReviewSubmittedHandler.java

```diff
@@ -83,7 +84,7 @@ private void handlePostHearingRefused(SscsCaseData caseData) {
         Long caseId = Long.valueOf(caseData.getCcdCaseId());
         PostHearing postHearing = caseData.getPostHearing();
         if (postHearing.isRefused()) {
-            if (isYes(postHearing.getSetAside().getRequestStatementOfReasons())) {
+            if (SET_ASIDE.equals(postHearing.getReviewType()) && isYes(postHearing.getSetAside().getRequestStatementOfReasons())) {
```

Comment:
Is this additional check strictly necessary? If the type is not set aside, then getSetAside() will generate a new blank SetAside, in which getRequestStatementOfReasons will be null.

Reply:
Adding this check for clarity and so i can push this ticket forward whilst i investigate the bug we had a call about